ui/js/models/repository.js

```diff
@@ -278,7 +278,7 @@ treeherder.factory('ThRepositoryModel', [
 
                                 // To get the current repo to display first, we must
                                 // ensure it's added to the array last, as per normal user interaction
-                                storedWatched = _.without(storedWatched, options.name);
+                                storedWatched = storedWatched.filter(function (value) { return value !== options.name; });
```

Comment:
Again, this could be:

    storedWatched.filter(value => (value !== options.name));